templates/profile-edit.html

```diff
@@ -81,22 +76,23 @@
                                 </div>
                             </div>
                             <!-- Save changes button-->
+
                             <div class="">
                                 <a href="{% url 'profile' user.id %}" class="btn btn-primary mx-1">
                                     Profile
                                 </a>
-                                <button class="btn btn-primary" type="button">Save changes</button>
+                                <button class="btn btn-primary save-button" type="submit">Save</button>
                             </div>
-                        </form>
+                        </div>
                     </div>
                 </div>
             </div>
-        </div>
+        </form>
     </div>
     <script>
         document.addEventListener("DOMContentLoaded",() => {
-            document.getElementById('imagefile').addEventListener('change', (event) => {
-                image = document.getElementById('imagefile')
+            document.getElementById('image-file').addEventListener('change', (event) => {
+                image = document.getElementById('image-file')
```

Comment:
Arent theese changes made in your another branch?
 [https://github.com/Aymammet/sky/pull/70]

Reply:
yes, the rest was done in another branch because it included post-edit file